lms/static/js/learner_dashboard/views/explore_new_programs_view.js

```diff
@@ -0,0 +1,44 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'text!../../../templates/learner_dashboard/explore_new_programs.underscore'
+           ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             exploreTpl
+         ) {
+            return Backbone.View.extend({
+                el: '.program-advertise',
+
+                tpl: _.template(exploreTpl),
+
+                initialize: function(data) {
+                    this.context = data.context;
+                    this.$parentEl = $(this.parentEl);
+
+                    if (this.context.xseriesUrl){
+                        // Only render if there is an XSeries link
+                        this.render();
```

Comment:
I call render in quite a lot of my view init functions. It is relatively unusual that you would initialize and then not immediately render said view. I like that it keeps the code a bit cleaner for instantiating a new view (no need to append ```.render()``` to the instantiation code). Because this view is so simple I cannot think of a use case where it would be instantiated and not immediately rendered.